tox.ini

```diff
@@ -36,5 +37,5 @@ setenv =
     TOXTEMPDIR={envtmpdir}
 commands =
   python -m pytest --cov=west {posargs:tests}
-  python -m flake8 --config={toxinidir}/tox.ini {toxinidir}
+  python -m flake8 --config={toxinidir}/tox.ini {toxinidir} # --color=never
```

Comment:
?

Reply:
This lets you click on the error messages in your IDE (emacs, sorry for outing you).

This is learned the hard way because this can't go in a config file for some unknown reason.

Reply:
Add this to your .emacs

```
(ignore-errors
  (require 'ansi-color)
  (defun my-colorize-compilation-buffer ()
    (when (eq major-mode 'compilation-mode)
      (ansi-color-apply-on-region compilation-filter-start (point-max))))
  (add-hook 'compilation-filter-hook 'my-colorize-compilation-buffer))
```

Reply:
Thanks, removed.